src/wrap.tsx

```diff
@@ -1,17 +1,18 @@
 import React from 'react'
 
 import { mockNetwork } from './mockNetwork'
-import { getConfig } from './config'
+import { getConfig, Mount } from './config'
+import { BrowserHistory, Response, Wrap, WrapOptions } from './models'
 
 beforeEach(() => {
   global.fetch = jest.fn()
 })
 
 afterEach(() => {
-  global.fetch.mockRestore()
+  ;(global.fetch as jest.MockedFunction<typeof fetch>).mockRestore()
```

Comment:
can we get rid of the semicolon?

Reply:
Prettier adds it to the save, I'm not sure why we need it.

Reply:
Its weird, but I guess it does not harm anybody. 

Another option will be to assign the `global.fetch.mockRestore` to a constant and then execute the `mockRestore` over it. But I don't know if I prefer that to the semicolon xD

Reply:
I'll check it. Thanks!

Reply:
```
afterEach(() => {
  const mockedFetch = global.fetch as jest.MockedFunction<typeof fetch>
  mockedFetch.mockRestore()
})
```
It works but I feel I'm hacking TS (as before, of course)

Reply:
indeed it doesn't feel right, let's keep the semicolon.

Reply:
Now, I don't know what I understand better 🤣.

Reply:
Let's keep the version using a constant, it looks more readable.